roscala/src/main/scala/coop/rchain/roscala/pools/SimpleStrandPool.scala

```diff
@@ -0,0 +1,33 @@
+package coop.rchain.roscala.pools
+import com.typesafe.scalalogging.Logger
+import coop.rchain.roscala.Vm.State
+import coop.rchain.roscala.ob.Ctxt
+
+import scala.collection.mutable
+
+class SimpleStrandPool extends StrandPool {
+  private val queue  = mutable.Buffer[Ctxt]()
+  private val logger = Logger("SimpleStrandPool")
+
+  override def append(task: Task): Unit = queue.append(task._1)
+
+  override def prepend(task: Task): Unit = queue.prepend(task._1)
+
+  override def getNextStrand(state: State): Boolean =
+    if (queue.isEmpty) {
+      logger.debug("Empty strandPool")
+      true
+    } else {
+      val ctxt = queue.remove(queue.size - 1)
+
+      logger.debug(s"Ctxt completed. Install $ctxt")
```

Comment:
I still find this irritating since the `Ctxt` that gave up control not necessarily *completed* (see the `(seq (foo actor) (foo actor))` example).
Please change to "Install $ctxt" or "Ctxt gave up control. Install $ctxt".